fmralign/pairwise_alignment.py

```diff
@@ -101,7 +98,12 @@ def fit_one_piece(X_i, Y_i, alignment_method):
                                        OptimalTransportAlignment,
                                        DiagonalAlignment)):
         alignment_algo = clone(alignment_method)
-    alignment_algo.fit(X_i, Y_i)
+    try:
+        alignment_algo.fit(X_i, Y_i)
+    except UnboundLocalError:
+        warn = "Unrecognized alignment method {}.".format(alignment_method) + \
+               " Please provide a recognized alignment method."
```

Comment:
Do we have a list of valid options to suggest ? Otherwise it's great like this

Reply:
I can't find a list... but restructuring so that there is one would make sense ! Might be something for a separate PR, though ? LMK what you think !

Reply:
I don't think it's a priority at all, thanks a lot for the test!